src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/DmnEvaluationService.java

```diff
@@ -130,25 +133,27 @@ private List<PermissionsDmnEvaluationResponse> performEvaluatePermissionsDmnActi
         }
     }
 
-    private Set<TaskTypesDmnResponse> performRetrieveTaskTypesDmn(String jurisdiction, String dmnNameField) {
+    private Set<TaskTypesDmnResponse> performRetrieveTaskTypesDmn(String jurisdiction, String dmnKeyLikeField) {
         try {
             List<TaskTypesDmnResponse> taskTypesDmnResponseList = camundaServiceApi.getTaskTypesDmnTable(
                 serviceAuthTokenGenerator.generate(),
                 jurisdiction.toLowerCase(Locale.ROOT),
-                dmnNameField
+                dmnKeyLikeField,
+                true
             );
```

Comment:
Would be good to log the keys and ids you found for debugging in future?

Reply:
will add a debug log